packages/anon-creds/src/AnonCredsApiOptions.ts

```diff
@@ -0,0 +1,16 @@
+import type { AnonCredsSchema } from './models/vdr'
+
+// FIXME: we should update the name in the AnonCreds registry interface to include extra wording in the name to prevent name clashes.
+export interface RegisterSchemaApiOptions {
+  schema: AnonCredsSchema
+
+  // Identifier of the identifier that will create the schema. In most cases a did, but can also be an URI.
+  //   TODO: do we want an issuerId? Or should we add such properties to an options object that is different per method
+  // more like the did registration interface?
+  issuerId: string
```

Comment:
Do we want to create a specific type for a DID or URI? something like `did:${string}:${string}` or would that just be an annoyance and not very useful?